auth-web/src/resources/display-mappers/invoice-status-display.ts

```diff
@@ -0,0 +1,20 @@
+import { InvoiceStatus } from '@/util/constants'
+
+export const invoiceStatusDisplay = {
+  [InvoiceStatus.APPROVED]: 'Processing',
+  [InvoiceStatus.CANCELLED]: 'Cancelled',
+  [InvoiceStatus.COMPLETED]: 'Completed',
+  [InvoiceStatus.CREATED]: 'Created',
+  [InvoiceStatus.CREDITED]: 'Credited',
+  [InvoiceStatus.DELETED]: 'Deleted',
+  [InvoiceStatus.DELETE_ACCEPTED]: 'Deleted',
+  [InvoiceStatus.PAID]: 'Completed',
+  [InvoiceStatus.PARTIAL]: 'Partial Paid',
+  [InvoiceStatus.PENDING]: 'Pending',
+  [InvoiceStatus.REFUNDED]: 'Refunded',
+  [InvoiceStatus.REFUND_REQUESTED]: 'Refund Requested',
+  // not sure about below
+  [InvoiceStatus.SETTLEMENT_SCHEDULED]: 'Processing',
+  [InvoiceStatus.UPDATE_REVENUE_ACCOUNT]: 'Completed',
```

Comment:
This is to update the revenue GL code usually for an invoice

Reply:
So this means it needs a GL code update? Or it had one already? Do you know what would make sense to show here for the description?